.buildkite/windows.rayci.yml

```diff
@@ -0,0 +1,52 @@
+group: ":windows: windows"
+steps:
+- label: ":windows: python tests"
```

Comment:
can we move these to individual team rayci (core, data, serve, ml, etc.)

Reply:
sure. moved.

in v1, all windows tests are in one group. hope that people can still find them.